helm/fiftyone-teams-app/values.yaml

```
@@ -23,6 +23,13 @@ apiSettings:
     # -- Logging level. Overrides the value of `FIFTYONE_ENV`.
     # Can be one of "DEBUG", "INFO", "WARN", "ERROR", or "CRITICAL".
     LOGGING_LEVEL: INFO
+
+  # -- Secret variables to be passed to the teams-api containers.
+  secretEnv: {}
+  # ENV_VAR_NAME:
+  #     secretName: the-pre-existing-secret
+  #     secretKey: the-key-in-the-secret
```

Spacing to allow for uncommenting via editors:

```suggestion
  # -- Secret variables to be passed to the teams-api containers.
  secretEnv: {}
  #   ENV_VAR_NAME:
  #     secretName: the-pre-existing-secret
  #     secretKey: the-key-in-the-secret
```